examples/generate-services-imperative/README.md

```diff
@@ -0,0 +1,27 @@
+# generate-services: Imperative Example
+
+### Overview
+
+This examples shows how to generate [Config Controller Service resources](https://cloud.google.com/config-connector/docs/reference/resource-docs/serviceusage/service) given a set Config Controller resources in the package by running [`generate-services`] function imperatively.
+
+### Fetch the example package
+
+Get the example package by running the following commands:
+
+```shell
+$ kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/generate-services-imperative
+```
+
+### Function invocation
+
+Invoke the function by running the following commands:
+
+```shell
+$ kpt fn eval generate-services-imperative --image gcr.io/kpt-fn/generate-services:unstable
```

Comment:
I'd suggest you to show users how to use `namespace` and `disable-on-destroy`.
And then in the `Expected result` section, you can mention that these settings have been reflected on the output resources.